.github/workflows/npm-published-simulation.yml

```diff
@@ -0,0 +1,103 @@
+# Instead of waiting for Yari to be published to npmjs.com and be upgraded
+# inside mdn/content by Dependabot, we do all those steps here using `npm pack`.
+
+name: NPM Publish simulation
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup Node.js environment
+        uses: actions/setup-node@v2.1.5
+        with:
+          node-version: "12"
+
+      - name: Cache node_modules
+        uses: actions/cache@v2.1.4
+        id: cached-node_modules
+        with:
+          path: |
+            node_modules
+          key: ${{ runner.os }}-${{ hashFiles('yarn.lock') }}
+
+      - name: Install all yarn packages
+        if: steps.cached-node_modules.outputs.cache-hit != 'true'
+        env:
+          PUPPETEER_SKIP_CHROMIUM_DOWNLOAD: 1
+        run: yarn --frozen-lockfile
+
+      - name: Set up a mock content repo
+        run: |
+          mkdir /tmp/content
```

Comment:
Would it be better to use `actions/checkout@v2` to checkout the actual `mdn/content` repo? Runs would be slightly slower, but might be worth getting a non-simulated run

Reply:
Yeah. It's not a bad idea! 
Perhaps I can 
1. clone it
2. `yarn add file:tarball`
3. `yarn install`

Reply:
Yeah, i think it would be something like
```yaml
    steps:
      - uses: actions/checkout@v2
        path: 'yari'
      - uses: actions/checkout@v2
        repository: 'mdn/content'
        path: 'content'
````

Reply:
I'm fine with keeping it at `mdn/content`. It works fine. I just cd into it.